models/channel.js

```diff
@@ -156,14 +156,19 @@ class Channel extends Model {
       logger.sensitive(`response = ${JSON.stringify(response) }`);
     }
 
-
     logger.sensitive(`record = ${JSON.stringify(this.record)}`);
     logger.sensitive(`data = ${JSON.stringify(this.data)}`);
     // logger.sensitive(`publicKey = ${JSON.stringify(response.publicKey)}`);
 
 
     if (this.accessLink) {
       return super.create(JSON.parse(gravity.decrypt(this.accessLink)));
+          // .then( channel  => {
```

Comment:
remove  unused code

Reply:
I left it there as a reminder to add the channel > sendFirstImage thing. I'll be implementing that today.